src/components/capture/creation/SchemaEditor.tsx

```diff
@@ -18,10 +25,16 @@ function NewCaptureEditor(props: NewCaptureEditorProps) {
     );
 
     const handlers = {
+        onChange: () => {
+            if (editorRef.current) {
+                setSchema(editorRef.current.getValue());
```

Comment:
Without testing this I thought the `getValue` call returned a strong and not a JSON object.